### PR TITLE
RAC-4005  Pylint cleanup in RackHD/test

### PR DESCRIPTION
**Pull request by jimturnquist**
- added mkcheck.sh script to perform pylint checking (and
  anything else we may eventually want to check).  This
  can be used by Jenkins as a code quality gate.
- Only actual errors are checked for ("pylint -E").  Cleaning
  up warnings is for another day.
- Updated a handful of errors that existed presently in RackHD/test

### CODE REVIEW

**Comment by jimturnquist** [comment]
Test this please.

**Comment by jimturnquist** [comment]
@johren @gpaulos @hohene @stuart-stanley @larry-dean

File: test/mkcheck.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    pylint_output_file="/dev/stdout"
+    copyright_output_file="/dev/stdout"
+elif [ $# -eq 1 ]; then
+    format="text"
+    pylint_output_file="/dev/stdout"
+    copyright_output_file="/dev/stdout"
+elif [ $# -eq 2 ]; then
+    PKG_VERSION=$1
+    format=$2
+    pylint_output_file="onserve_${PKG_VERSION}_pylint.${format}"
```
Comment (gpaulos) [inline test/mkcheck.sh]: Do we want to continue using 'onserve' nomenclature in the code?

**Comment by jimturnquist** [comment]
Pulling out for now...  I want to rethink mkcheck.sh a bit more.

**Pull request closed without merging**